boards/arm/nrf52833dk_nrf52820/nrf52833dk_nrf52820.dts

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52820_qdaa.dtsi>
+
+/ {
+	model = "Nordic nRF52833 DK NRF52820";
+	compatible = "nordic,nrf52833-dk-nrf52820", "nordic,nrf52820-qdaa",
```

Comment:
There are some recent changes related to `compatible` here: https://github.com/zephyrproject-rtos/zephyr/pull/24774.